datarepo-clienttests/src/main/java/scripts/testscripts/SnapshotScaleDelete.java

```diff
@@ -0,0 +1,98 @@
+package scripts.testscripts;
+
+import bio.terra.datarepo.api.RepositoryApi;
+import bio.terra.datarepo.client.ApiClient;
+import bio.terra.datarepo.model.DeleteResponseModel;
+import bio.terra.datarepo.model.JobModel;
+import bio.terra.datarepo.model.SnapshotModel;
+import bio.terra.datarepo.model.SnapshotSummaryModel;
+import java.util.ArrayList;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import runner.config.TestUserSpecification;
+import scripts.testscripts.baseclasses.SimpleDataset;
+import scripts.utils.DataRepoUtils;
+
+public class SnapshotScaleDelete extends SimpleDataset {
+  private static final Logger logger = LoggerFactory.getLogger(SnapshotScaleDelete.class);
+  private int numSnapshots = 1;
+  private List<SnapshotSummaryModel> snapshotList = new ArrayList<>();
+
+  /** Public constructor so that this class can be instantiated via reflection. */
+  public SnapshotScaleDelete() {}
+
+  public void setParameters(List<String> parameters) {
```

Comment:
This class has a number of overridden methods but doesn't use `@Override` annotations. It looks like this is typical for the code in clienttests, but they should really have this annotation. The IDE can auto-fix this for you.

Reply:
Hmmm IntelliJ says it can't find any overridden methods 🤔

Reply:
That's odd, I checked out your branch and intellij is highliting them for me. The Editor Inspection is called `Java | Inheritance issues | Missing @Override annotation` and I can see them in the editor if I change Severity to Warning. By default the setting is No highliting, only fix.